lib/_http_incoming.js

```diff
@@ -67,6 +69,8 @@ IncomingMessage.prototype.setTimeout = function(msecs, callback) {
 
 
 IncomingMessage.prototype.read = function(n) {
+  if (!this._consuming)
+    this._readableState.readingMore = false;
```

Comment:
Okay, does that mean that in practice `this._readableState.readingMore` can be false when `this._consuming` is true?

A comment in the source explaining all that would definitely be appreciated. :-)